test/utils/time.ts

```diff
@@ -0,0 +1,50 @@
+import { ethers } from "hardhat";
```

Comment:
Creating these functions was a great idea.

Reply:
lol thanks for the shout out -- i'm sure i cribbed some version of this from elsewhere on the internet though ;)

we're all standing on the shoulders of giants